x-pack/plugins/alerting/server/usage/alerts_usage_collector.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
+import { get } from 'lodash';
+import { TaskManagerStartContract } from '../../../task_manager/server';
+import { AlertsUsage } from './types';
+
+export function createAlertsUsageCollector(
+  usageCollection: UsageCollectionSetup,
+  taskManager: TaskManagerStartContract
+) {
+  return usageCollection.makeUsageCollector({
+    type: 'alerts',
+    isReady: () => true,
+    fetch: async (): Promise<AlertsUsage> => {
+      try {
+        const doc = await getLatestTaskState(await taskManager);
+        // get the accumulated state from the recurring task
+        const state: AlertsUsage = get(doc, 'state');
+
+        return {
+          ...state,
+        };
+      } catch (err) {
+        return {
+          count_total: 0,
+          count_active_total: 0,
+          count_disabled_total: 0,
+          throttle_time: {
+            min: 0,
+            avg: 0,
+            max: 0,
+          },
+          schedule_time: {
```

Comment:
After checking off line with @YulNaumenko , the mapping for the `schedule_time` and `throttle_time` properties should be strings.